product-service/src/functions/hello/handler.ts

```diff
@@ -0,0 +1,14 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
```

Comment:
Was "hello" function committed by mistake? Guess, we don't have this requirement in Task 3

Reply:
fixed in pr for Task 4